deployments/liqo/templates/liqo-clusterid-configmap.yaml

```diff
@@ -9,10 +9,15 @@ metadata:
     {{- include "liqo.labels" $clusterIdConfig | nindent 4 }}
   name: {{ include "liqo.prefixedName" $clusterIdConfig }}
 data:
-  {{- if (not $oldObject) }}
-  CLUSTER_ID: {{ uuidv4 }}
-  {{- else }}
+  {{- if $oldObject }}
+  {{- if and (.Values.discovery.config.clusterIDOverride) (ne $oldObject.data.CLUSTER_ID .Values.discovery.config.clusterIDOverride) }}
+    {{ fail "Changing the cluster ID by providing a different .discovery.config.clusterIDOverride value is not allowed." }}
+  {{- end}}
   CLUSTER_ID: {{ $oldObject.data.CLUSTER_ID }}
+  {{- else if .Values.discovery.config.clusterIDOverride }}
```

Comment:
Before merging, I was wondering whether it might be appropriate to add a regex check to ensure this actually looks like a UUID. @QcFe, @aleoli WDYT?

Reply:
Not a problem for me

Reply:
👍